### ISSUE TITLE
nus_wide MAP ?

### ISSUE THREAD

**Issue by xy4869** [yuanli2333/Hadamard-Matrix-for-hashing#6]
May I know your specific parameters in NUS_ Wide？I use your library's parameters to run out the result is only 0.8188 using resnet model and 64 bit hash code in  NUS_ Wide dataset

**Comment by swuxyj** [comment]
I get mAP 0.844 when use
`
optimizer = torch.optim.RMSprop(params_list, lr = 1e-5, weight_decay=10 ** -5)
`

**Comment by yuanli2333** [comment]
> I get mAP 0.844 when use
> `optimizer = torch.optim.RMSprop(params_list, lr = 1e-5, weight_decay=10 ** -5)`

Nice try!

**Comment by Kenny-Li2023** [comment]
> May I know your specific parameters in NUS_ Wide？I use your library's parameters to run out the result is only 0.8188 using resnet model and 64 bit hash code in NUS_ Wide dataset

Can you share the NUS_WIDE dataset with me

**Comment by swuxyj** [comment]
> > May I know your specific parameters in NUS_ Wide？I use your library's parameters to run out the result is only 0.8188 using resnet model and 64 bit hash code in NUS_ Wide dataset
> 
> Can you share the NUS_WIDE dataset with me

NUS_WIDE dataset can be downloaded in [HashNet](https://github.com/thuml/HashNet/tree/master/pytorch) or [DSDH_PyTorch](https://github.com/TreezzZ/DSDH_PyTorch)

**Comment by Kenny-Li2023** [comment]
> > > May I know your specific parameters in NUS_ Wide？I use your library's parameters to run out the result is only 0.8188 using resnet model and 64 bit hash code in NUS_ Wide dataset
> > 
> > 
> > Can you share the NUS_WIDE dataset with me
> 
> NUS_WIDE dataset can be downloaded in [HashNet](https://github.com/thuml/HashNet/tree/master/pytorch) or [DSDH_PyTorch](https://github.com/TreezzZ/DSDH_PyTorch)

Thanks！！The format of the NUS_WIDS dataset is different from the script format in the code. How do I generate my own script 
please?

**Comment by swuxyj** [comment]
> > > > May I know your specific parameters in NUS_ Wide？I use your library's parameters to run out the result is only 0.8188 using resnet model and 64 bit hash code in NUS_ Wide dataset
> > > 
> > > 
> > > Can you share the NUS_WIDE dataset with me
> > 
> > 
> > NUS_WIDE dataset can be downloaded in [HashNet](https://github.com/thuml/HashNet/tree/master/pytorch) or [DSDH_PyTorch](https://github.com/TreezzZ/DSDH_PyTorch)
> 
> Thanks！！The format of the NUS_WIDS dataset is different from the script format in the code. How do I generate my own script
> please?

You can refer to this [code](https://github.com/swuxyj/DeepHash-pytorch/blob/master/data/nuswide_21/code.py)  to divide your own dataset

**Comment by Kenny-Li2023** [comment]
Why is the accuracy I trained is only 0.7146 on the NUS_WIDE dataset?

**Comment by Kenny-Li2023** [comment]
> I get mAP 0.844 when use
> `optimizer = torch.optim.RMSprop(params_list, lr = 1e-5, weight_decay=10 ** -5)`

你好，请问你具体改了哪些参数？我直接用作者的代码但是精确度只有0.7146，论文用的NUS_WIDE不是原始NUS_WIDE数据集吧？
